src/storage/hummock_sdk/src/slice_transform.rs

```diff
@@ -215,10 +219,94 @@ impl SliceTransform for MultiSliceTransform {
     }
 }
 
+#[derive(Default)]
+pub struct SliceTransformManagerInner {
+    table_id_to_slice_transform: RwLock<HashMap<u32, SliceTransformImpl>>,
```

Comment:
`RwLock<HashMap<u32, SliceTransformImpl>>` -> `RwLock<HashMap<u32, Arc<SliceTransformImpl>>>`?

Reply:
use Mutex to impletement